### DISCUSSION TITLE
Can't search for sve_bitwise_unpredicate_logical

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#20750] [Other Feature Feedback, Questions, & Ideas]
## Expected results

I'm trying to find this:
https://github.com/Linaro/vixl/blob/ece47d8a023f4d73c21ab4f55eba59da59a69c9b/test/aarch64/test-assembler-sve-aarch64.cc#L482

## Steps

I'd like this to work:
https://cs.github.com/Linaro/vixl?q=sve_bitwise_unpredicate_logical

This also doesn't work:
https://github.com/Linaro/vixl/search?q=sve_bitwise_unpredicate_logical

One can search for the various terms:
https://cs.github.com/Linaro/vixl?q=sve+bitwise+unpredicate+logical

In theory, this should do the right thing:
https://cs.github.com/Linaro/vixl?q=sve+bitwise+unpredicate+logical+path%3Atest%2Faarch64%2Ftest-assembler-sve-aarch64.cc

This doesn't work either:
https://cs.github.com/Linaro/vixl?q=the+path%3Atest%2Faarch64%2Ftest-assembler-sve-aarch64.cc

Which indicates that the file isn't being indexed.

If I ask cs about an unindexed file, at the very least, cs should tell me:
* it isn't indexed
* why it isn't indexed
* what I can do instead

**Comment by look** [comment]
Thanks for the feedback. As [documented in the FAQ](https://cs.github.com/about/faq#indexed-content) extremely large files like this one are not indexed. I agree we should do a better job of indicating that when searching for a path like in your example.
